ansi/cwd.go

```diff
@@ -0,0 +1,21 @@
+package ansi
+
+import (
+	"net/url"
+)
+
+// NotifyWorkingDirectory returns a sequence for notifying the program's
+// current working directory.
+//
+//	OSC 7 ; Pt BEL
+//
+// Where Pt is a URL in the format "file://[host]/[path]".
+// Set host to "localhost" if this is a path on the local computer.
+func NotifyWorkingDirectory(host string, path string, params ...string) string {
```

Comment:
`params` is not used
```suggestion
func NotifyWorkingDirectory(host string, path string) string {
```

Reply:
Oops, thanks for catching this.